src/commands/plugins/install.ts

```diff
@@ -195,6 +195,12 @@ Use the <%= config.scopedEnvVarKey('NPM_REGISTRY') %> environment variable to se
         throw error
       }
 
+      const pluginInstance = new Plugin(plugin)
+      await pluginInstance.load()
+      this.config.plugins.set(pluginInstance.name, pluginInstance)
+
+      await this.config.runHook('plugins:postinstall', {})
```

Comment:
plugin-autocomplete needs `config.plugins` with the new plugin populated for it to be included in the refreshed autocomplete files:
https://github.com/oclif/plugin-autocomplete/blob/986c6ec238ff3764c78331de7ea7b49e0670a47c/src/commands/autocomplete/create.ts#L95